src/content/docs/en/tutorial/1-setup/1.mdx

```diff
@@ -28,18 +28,18 @@ You can access the command line through a local terminal program for your operat
 
 ### Node.js
 
-For Astro to run on your system, you will also need to have [**Node.js**](https://nodejs.org/en/) installed, version `v18.17.1` or `v20.3.0` or later. (`v19` is not supported.)
+For Astro to run on your system, you will also need to have [**Node.js**](https://nodejs.org/en/) installed, version `v18.20.8` or `v20.3.0`, `v22.0.0` or higher. (`v19` and `v21` are not supported.)
 
 To check to see whether you already have a compatible version installed, run the following command in your terminal:
 
 ```sh
 node -v
 
 // Example output
-v18.17.1
+v18.20.8
 ```
 
-If the command returns a version number higher than `v18.17.1` or `v20.3.0` (excluding any `v19`), you're good to go!
+If the command returns a version number higher than `v18.20.8` or `v20.3.0` (excluding any `v19` or `v21`), you're good to go!
```

Comment:
```suggestion
If the command returns a version number supported by Astro, you're good to go!
```
Again, I don't think we can say this simply here like we could before. So maybe this is sufficient?